src/converters/replugged.ts

```diff
@@ -1,13 +1,45 @@
+import { ClassDeclaration, ClassMethod, ExportDefaultDeclaration, Identifier, Statement } from "@babel/types";
 import { IModImplementation } from "../api/ModImplementation.js";
 import { createFunctionFromObjectProperty } from "../api/RuntimeGenerators.js";
 import { IBaseWebpackApi } from "../api/Webpack.js";
+import { parse } from "@babel/parser";
 
 class RPWebpackApi implements IBaseWebpackApi {
     get getModule() {
         return createFunctionFromObjectProperty("replugged.webpack", "getModule");
     }
 }
 
+export function convertFormat(ast: Statement[]) {
+    let targetClassName = undefined;
+    for (const astNode of ast) {
+        if (astNode.type !== "ExportDefaultDeclaration") continue;
+        const exportDeclaration = astNode as ExportDefaultDeclaration;
+        if (exportDeclaration.declaration.type !== "ClassDeclaration") continue;
+        const exportedClass = exportDeclaration.declaration as ClassDeclaration;
+        const hasStartMethod = exportedClass.body.body.some(
+            (method) => ((method as ClassMethod).key as Identifier)?.name === "start",
+        );
+        if (!hasStartMethod) continue;
+        targetClassName = exportedClass.id?.name;
+        break;
+    }
+    if (targetClassName != undefined) {
+        // const instanceVar = variableDeclaration("const", [
```

Comment:
Is this part still needed?

Reply:
I'm not sure. At first I wanted to make the code injection using Babel methods for AST creation but then I noticed thats too much work, so it's here.

Reply:
Oke, good for me @zrodevkaan wanna take a look at the pr before merging?